### ISSUE TITLE
Mouse pointer y-offset

### ISSUE THREAD

**Issue by tamasgal** [koekeishiya/yabai#2181]
I cannot reproduce this bug but it happens multiple times a day. I only noticed it when using an external monitor (via a DisplayLink driver on my MacBook Air M1). The mouse pointer is off by approximately 150px in the y direction, so that visually the click and point action is about 150px above the actual shown position.

I managed to record a video (see below).

I am using `yabai-v6.0.15` but will now updated to the latest master and report back if I encounter this again.

https://github.com/koekeishiya/yabai/assets/1730350/0a4778a1-6023-4220-988e-5b754652f4a4

**Comment by tamasgal** [comment]
I think this is related to Safari. I just noticed that it happens only in the tab where [Pluto.jl](https://github.com/fonsp/Pluto.jl) is running. Now I recall that I mostly see this on website with a (rich) text input, like our GitLab instance.

I am not sure if this is related to yabai but I only encountered these issues since I use yabai 😕

Btw. I just updated to yabai 7.0.2 and the Safari tab with the mouse glitch still has the offset, but again, only that very single tab. If I reload it, it's gone.

Let me leave this issue open for a day or two, in case someone has observed something similar.
